thoth-core/src/main/java/fr/maif/eventsourcing/impl/KafkaEventPublisher.java

```diff
@@ -163,6 +165,9 @@ public KafkaEventPublisher(ActorSystem system, ProducerSettings<String, EventEnv
 
     @Override
     public void close() throws IOException {
+        if(Objects.nonNull(killSwitch)) {
```

Comment:
Used for testing + maybe it can be usefull in some cases

Reply:
With `control` you can do the same non ?